dev_guide/integrating_external_services.adoc

```diff
@@ -53,12 +53,20 @@ ignore the service and allow you to specify endpoints manually:
 ----
 {
   "kind": "Service",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
   "metadata": {
     "name": "example-mysql-service"
   },
-  "spec": { <1>
```

Comment:
you removed the link to the footnote.  though i'm not sure what that footnote is trying to say anyway.